src/opus.py

```diff
@@ -173,11 +197,26 @@ async def load_nodes(nodes_dict: Dict[str, Any], script_path: Path) -> Dict[str,
             if next_pair is None:
                 break
     
+    # Convert nodes from dicts to Node objects, and pick out all inlined actions.
+    # We make up action IDs.
     nodes = {}
+    action_dicts = {}
     for key, node in nodes_dict.items():
         typed_node = node.copy()
         if isinstance(typed_node.get("next"), list):
-            typed_node["next"] = [NodeChoice(
-                **node_choice) for node_choice in node["next"]]
+            for i, choice in enumerate(typed_node["next"]):
+                if isinstance(choice.get("actions"), list):
```

Comment:
Isnt this always a list? But it could either be a list of strings (aka action IDs) or a list of inlined actions?

Reply:
It could also be None, since it's optional. But this is not the clearest way to write this, so I'll change it it to ```if "actions" in choice``` (and fix a bug I found).